tardis/plasma/properties/atomic.py

```diff
@@ -108,8 +109,17 @@ def calculate(self, atomic_data, selected_atoms):
 class IonizationData(ProcessingPlasmaProperty):
     name = 'ionization_data'
 
-    def calculate(self, atomic_data, selected_atoms):
+    def calculate(self, atomic_data, levels):
         if self.value is not None:
             return self.value
         else:
             return atomic_data.ionization_data
+
+class ZetaData(ProcessingPlasmaProperty):
+    name = 'zeta_data'
```

Comment:
There aren't any underscores in the dataset I used for my paper, but there are in the testing one. Just for `_lines` and `_levels` from what I can see. Would you like me to change that and make a PR to the Tardis master branch? I think I know how (I modified an atomic dataset before to update the helium data). I can try to update the `ZetaData` stuff too?